src/action-types.js

```diff
@@ -1,8 +1,8 @@
+import actionTypesWithArray from './action-types-array';
+
 export default function actionTypes(namespace, ...constants) {
-  return Object.freeze(
-    constants.reduce((obj, constant) => {
-      obj[constant] = `${namespace}/${constant}`;
-      return obj;        
-    }, {})
-  )
+  console.warn(
```

Comment:
I don't want to warn for this.  It'll work fine as-is for this release.  We'll wait until later to break it and release a new major version.